File: workspace/1_FunnyCase/test/strings/FunnyCaseTest.java
package strings;

import static junit.framework.Assert.assertEquals;

import org.junit.Test;


public class FunnyCaseTest {
	
	@Test
	public void emptyString() {
		assertEquals("", StringUtil.funnyCase(""));
	}
	
	@Test
	public void someString() {
		assertEquals("FuNnY cAsE", StringUtil.funnyCase("Funny Case"));
	}

}

File: solution/1_FunnyCase/1_FunnyCase/FunnyCaseTest.cs
using System;
using System.Text;
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using strings;

namespace test
{
    [TestClass]
    public class FunnyCaseTest
    {
        [TestMethod]
        public void emptyString() {
		    Assert.AreEqual("", StringUtil.funnyCase(""));
	    }

        [TestMethod]
	    public void someString() {
            Assert.AreEqual("FuNnY cAsE", StringUtil.funnyCase("Funny Case"));
	    }
    }
}
